decidim-admin/spec/features/participatory_process_admin_manages_participatory_process_admins_spec.rb

```diff
@@ -25,6 +25,7 @@
       switch_to_host(organization.host)
       login_as user, scope: :user
       visit decidim_admin.participatory_process_path(participatory_process)
+      click_link "Process admins"
```

Comment:
This looks too fragile. Maybe you could create a new helper `within_process_menu` (only in admin/support) to ensure that link is where it should be and you don't break the test if a similar named link appears in the future.